Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,18 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+// - In `exercise.js` write a program that will take the `hogwarts` array as input and display the names of the people who belong to the Gryffindor house.
+// - Use object destructuring to extract the values you need out of each element in the array.
+function gryffindorMembers(hogwarts) {
+  return hogwarts.filter(({ house }) => house === "Gryffindor")
+    .map(({ firstName, lastName }) => `${firstName} ${lastName}`);
+}
+console.log(gryffindorMembers(hogwarts));
```

Comment:
Suggestion (non-blocking): I would output text explaining what the output is. 
`console.log('GryffindorMembers are:', gryffindorMembers(hogwarts))`.